tests/app/celery/test_process_ses_receipts_tasks.py

```diff
@@ -616,80 +614,6 @@ def get_complaint_notification_and_email(mocker):
     return complaint, notification, recipient_email
 
 
-class TestSendEmailStatusToVAProfile:
```

Comment:
- Moved these to own test module to reflect refactor 
- Renamed and moved to `test_send_va_profile_notification_status_tasks`